dask_geomodeling/geometry/field_operations.py

```diff
@@ -753,7 +753,7 @@ def choices(self):
 
     @staticmethod
     def process(source, *choices):
-        result = pd.Series(np.nan, index=source.index)
+        result = pd.Series(np.nan, dtype=object, index=source.index)
```

Comment:
Why did you touch this?

I guess it originally was `float`; reading the docstring I think this geoblock is also intended for numbers. Changing it to `object` will allow any type, also strings, is that what something you intend to fix? For floats and ints this will be a suboptimal efficiency...

Reply:
`FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value '['A']' has dtype incompatible with float64, please explicitly cast to a compatible dtype first. `

I used this solution from the other open PR, it solves the warning and I didn't dive into it further.

Reply:
I fixed the rtd action somehow, but I don't know where to find the warning that you mention. I will merge for now and then we move on.